Adamant/Modules/Delegates/DelegatesListViewController.swift

```diff
@@ -242,7 +242,8 @@ final class DelegatesListViewController: KeyboardObservingViewController {
         }
         
         bottomPanel.snp.makeConstraints {
-            $0.leading.trailing.bottom.equalTo(view.safeAreaLayoutGuide)
+            $0.bottom.equalTo(view.safeAreaLayoutGuide)
+            $0.leading.trailing.equalToSuperview().inset(10)
```

Comment:
It's redundant, this panel is ok with full width

Reply:
`leading.trailing` -> `horizontal`